dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.anno-2.0.feature

```diff
@@ -17,6 +17,6 @@ IBM-Process-Types: server, \
 -bundles=com.ibm.ws.anno
 -jars=com.ibm.websphere.appserver.spi.anno; location:=dev/spi/ibm/
 -files=dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.anno_1.1-javadoc.zip
-kind=noship
+kind=beta
 edition=full
```

Comment:
In addition to fixing the personal build failures, I also think `edition` should be set to `core`?

Reply:
Good catch @tevans78 I verified the javaee versions of the features / previous features for the same functionality were all in the `core` edition rather than the `full` edition. Not sure why these were originally added to `full` but I updated them now.